recipes/uv/bld.bat

```diff
@@ -0,0 +1,24 @@
+@echo on
+
+cd crates\uv
+
+cargo install ^
+    --locked ^
+    --path . ^
+    --profile release ^
+    --root "%PREFIX%" ^
+    || exit 1
+
+md "%PREFIX%\Scripts" ^
+    || echo "`%PREFIX%\Scripts` already exists"
+
+move "%PREFIX%\bin\uv.exe" "%PREFIX%\Scripts\uv.exe" ^
```

Comment:
What's the reason you put this here? Normally, we would place it in `%LIBRARY_PREFIX%\bin`